index.js

```diff
@@ -4,9 +4,8 @@ import fs from 'graceful-fs';
 import {xdgConfig} from 'xdg-basedir';
 import writeFileAtomic from 'write-file-atomic';
 import dotProp from 'dot-prop';
-import uniqueString from 'unique-string';
 
-const configDirectory = xdgConfig || path.join(os.tmpdir(), uniqueString());
+const configDirectory = xdgConfig || fs.mkdtempSync(fs.realpathSync(os.tmpdir()) + path.sep);
```

Comment:
Missing here:

- [ ] Only make the dinner on instantiation, not when the module is loaded